README.md

```diff
@@ -546,6 +546,30 @@ ort:
     storageWriters: ["artifactoryStorage"]
 ```
 
+### AWS S3 Storage
+
+An AWS S3 Bucket can be used to store scan results. You must provide a previously created Bucket, an optional AWS region
+(if not present it will use [default](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/region-selection.html))
+and credentials (if not provided, system [default](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/credential-providers.html)
+values will be used). Optionally, you can set the data to be stored compressed.
+
+```yaml
+ort:
+  scanner:
+    storages:
+      awsStorage:
```

Comment:
AWS offers multiple storage services see https://aws.amazon.com/products/storage/ therefore I believe it's better to not name the key `awsStorage` in case we want to add other AWS storage service in the future.

What do you think of renaming `awsStorage` to `awsS3Storage` across this PR?